test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -197,6 +208,71 @@ def test_emacs(image):
     _run_cmd_on_container(container_name, ctx, "emacs -version")
 
 
+@pytest.mark.model("N/A")
+def test_cuda_paths(gpu):
+    """
+    Test to ensure directory structure for GPU files has cuda version in it
+
+    :param gpu: gpu image uris
+    """
+    image = gpu
+    if "example" in image:
+        pytest.skip("Skipping example containers which are not explicitly tied to a cuda version")
+
+    dlc_path = os.getcwd().split('/test/')[0]
+    job_type = "training" if "training" in image else "inference"
+
+    # Ensure that image has a supported framework
+    frameworks = ("tensorflow", "pytorch", "mxnet")
+    framework = ''
+    for fw in frameworks:
+        if fw in image:
+            framework = fw
+            break
+    assert framework, f"Cannot find any frameworks {frameworks} in image uri {image}"
+
+    # Get cuda and framework version through regex
+    cuda_version_match = re.search(r'-cu\d+-', image)
+    cuda_version = cuda_version_match.group().strip('-')
+
+    framework_version_match = re.search(r':\d+.\d+.\d+', image)
```

Comment:
nit: Can use regular expression `r":(\d+(.\d+){2})"`, and get the value of `fw_version_match.group(1)`